wiki/Tournaments/RMoT_Invitational/2/en.md

```diff
@@ -1,14 +1,14 @@
-# RMoT Invitationals 2
+# RMoT Invitational 2
 
-The Rapid Monthly osu! Tournament Invitationals \#2 is a tournament organized and ran by the Rapid Tournaments team.
+The Rapid Monthly osu! Tournament Invitational \#2 is a tournament organized and ran by the Rapid Tournaments team.
 It is an exclusive tournament where only players who reached semi-finals in one of the last 4 RMoT's get invited.
 
-The tournament brackets can be found [on Challonge.](https://challonge.com/rmotinvitational2)
+The tournament brackets can be found [on Challonge.](https://challonge.com/rmotinvitational2) Most of the information here complies with the [forum post](https://osu.ppy.sh/forum/t/589111).
 [![][flag_PL]MagBlood](https://osu.ppy.sh/u/6178640) won the Invitational, beating [![][flag_PL]bsurma](https://osu.ppy.sh/u/509575) in the finals. 
```

Comment:
i hate those flags in links; at least give a space between flag link and username 🤔